scripts/build-website.sh

```diff
@@ -14,7 +14,7 @@ if [[ $SKIP_DOWNLOADS != "true" ]]; then
   ./scripts/download-releases.sh --major 1
   ./scripts/download-releases.sh --major 2
   ./scripts/download-releases.sh --major 3
-  ./scripts/download-releases.sh --minor 4
+  ./scripts/download-releases.sh --major 4
```

Comment:
Do you think we can restore this value to `minor` now?

Reply:
From memory perspective it should be fine now, so I think you can give it a try.